pkg/crc/config/settings_test.go

```diff
@@ -158,3 +159,19 @@ func TestUnsetPreset(t *testing.T) {
 		IsSecret:  false,
 	}, cfg.Get(CPUs))
 }
+
+func TestPath(t *testing.T) {
+	cfg, err := newInMemoryConfig()
+	require.NoError(t, err)
```

Comment:
A test for the default value would be useful imo as this was initially buggy.
Something like this should do the trick:

```
	assert.Equal(t, SettingValue{
		Value:     Path(""),
		Invalid:   false,
		IsDefault: true,
		IsSecret:  false,
	}, cfg.Get(ProxyCAFile))
```